<!DOCTYPE html>
<html lang="en">
<head>
    <title>LiuLab Publications</title>
    <meta charset="UTF-8">
    <meta name="description" content="北京大学, 刘涛课题组, LiuLab, PKU, Tao Liu">
    <meta name="keywords" content="LiuLab, 刘涛, Tao liu, PKU, 北京大学, Peking University">
    <meta name="copyright" content="LiuLab">
    <base href="">
    <link rel="stylesheet" type="text/css" href="css/Homepage.css">
</head>
<body>
<!-- Back to top -->
<button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
<!-- Back to top End -->
<!-- Header -->
<div id="header-container"></div>
<!-- Header End -->
<div class="content-container">
    <!-- Publication Sidebar -->
    <div class="publication-sidebar">
        <a class="publication-sidebar-item-active tablink" onclick="openYear(event, 'After PKU')">After PKU</a>
        <a class="publication-sidebar-item tablink" onclick="openYear(event, 'Before PKU')">Before PKU</a>
    </div>
    <!-- Publication Sidebar End -->
    <!-- Publications -->
    <div style="margin-left: 20%; margin-right: 10%">
        <ol>
		<!-- After PKU -->
        <div class="publications" id="After PKU">
            <h2>After PKU</h2>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/10.1021/acs.nanolett.4c06362"
                   target="_blank"> Zeng, Z.<sup>#</sup>, Cai, W.<sup>#</sup>, ... , Wang, Y.* Liu, T.*(2025). Small Molecule Drugs Triggered the Activation of Macrocycle Masked Proteins
                    <i>Nano. Lett.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2025_nanoletters.jpeg" height="200px"
                         alt="Figure">
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/10.1021/acs.chemrev.4c00216"
                   target="_blank"> Huang, Y.<sup>#</sup>, Zhang, P.<sup>#</sup>, Wang, H.<sup>#</sup>, ... , Liu, T.* Luo, X.*(2024). Genetic Code Expansion: Recent Developments and Emerging Applications
                    <i>Chem. Rev.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2024_chemicalreview.jpeg" height="200px"
                         alt="Figure">
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://www.nature.com/articles/s41557-024-01675-x"
                   target="_blank"> Hu, L.<sup>#</sup>, Cao, W.<sup>#</sup>, ... , Lou, X.* Liu, T.* (2024). Designing Artificial Fluorescent Proteins and Biosensors by Genetically Encoding Molecular Rotor-based 
                   Amino Acids
                    <i>Nat. Chem.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2024_natchem.jpeg" height="200px"
                         alt="Figure">
                </div>
            </div>

            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/jacs.4c07958"
                   target="_blank"> Li, Y.<sup>#</sup>,Su, Y.<sup>#</sup>, ... , Lou, X.* Peng, Q.* Liu, T.* (2024). Computation-Guided Discovery of Diazole Monosubstituted Tetrazines as Optimal Bioorthogonal Tools. <i>J. Am. Chem. Soc.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2024_jacs.jpeg" height="200px"
                         alt="Figure">
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://www.sciencedirect.com/science/article/pii/S1001841723008744?via%3Dihub"
                   target="_blank"> Dai, Z., Tan, L., ... , Liu, T.* (2024). Site-specific incorporation of reduction-controlled guest amino acids into proteins for cucurbituril recognition
                    <i>Chin. Chem. Lett.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2024_chinachemlett.jpg" height="200px"
                         alt="Figure">
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://www.nature.com/articles/s41589-023-01533-w"
                   target="_blank"> Hao, M.<sup>#</sup>,Ling, X.<sup>#</sup>, Sun, Y.<sup>#</sup>, ... , Liu, T.* (2024). Tracking endogenous proteins based on RNA editing-mediated genetic code expansion. <i>Nat. Chem. Biol.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2024_ncb.jpeg" height="200px"
                         alt="Figure">
                </div>
            </div>

            <div class="publication">
                <li><h3>
                    <a href="https://doi.org/10.1016/j.chempr.2023.08.003"
                   target="_blank"> Quan, M., Liu, T. (2023). In memory of Professor Wei Jiang, a rising star in supramolecular chemistry. <i>Chem.</i></a>
                </h3></li>
            </div>

            <div class="publication">
                <li><h3>
                    <a href="https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202200716"
                   target="_blank"> Wu, D., Liu, T.* (2023). Studying Reversible Protein Post-translational Modification through Co-translational Modification. <i>Chembiochem.</i></a>
                </h3></li>
            </div>

            <div class="publication">
                <li><h3>
                    <a href="https://www.chinesechemsoc.org/doi/10.31635/ccschem.023.202202471"
                   target="_blank"> Ling, X., Zuo, Y., ... , Liu, T.* (2023). Genetic Encoding of a Photocaged Glutamate for Optical Control of Protein Functions. <i>CCS Chemistry.</i></a>
                </h3></li>
                <div class="recent-publication-picture">
                    <img src="new pics/CCS Chem 2021.jpg" height="200px"
                         alt="Figure">
                </div>
            </div>

            <div class="publication">
                <li><h3>
                    <a href="https://link.springer.com/protocol/10.1007/978-1-0716-3251-2_6"
                   target="_blank"> Wang, Y., Cai, W., Han, B., Liu, T.* (2023). Protein Expression with Biosynthesized Noncanonical Amino Acids. <i>Methods Mol Biol.</i></a>
                </h3></li>
            </div>

            <div class="publication">
              <li><h3><a href="https://doi.org/10.1016/j.chempr.2023.05.034"
                   target="_blank"> Cao, W.<sup>#</sup>, Wang, H.<sup>#</sup>, Quan, M.<sup>#</sup>,..., Jiang, W. & Liu, T.* (2023). Reversible control of tetrazine bioorthogonal reactivity by naphthotube-mediated host-guest recognition. <i>Chem.</i></a>
              </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2023 Chem.jpg" height="200px"
                         alt="Figure">
                </div>
            </div>
            <div class="publication">
                <li><h3><a href="https://www.nature.com/articles/s41467-023-36658-y"
                    target="_blank"> Wang, Y.<sup>#</sup>, Zhang, J.<sup>#</sup>, Han, B.<sup>#</sup>,...,Yang, X.* & Liu, T.* (2023). Noncanonical amino acids as doubly bio-orthogonal handles for one-pot preparation of protein multiconjugates. <i>Nature Communications.</i></a>
                </h3></li>
                    <div class="recent-publication-picture">
                        <img src="images/2023 N Com.png" height="200px"
                            alt="Figure">
                    </div>
                </div>
            <div class="publication">
                <li><h3><a href="https://onlinelibrary.wiley.com/doi/10.1002/anie.202205570"
                    target="_blank"> Wu, D.<sup>#</sup>, Zhang, Y.<sup>#</sup>, Tang Z.<sup>#</sup>, Chen X.<sup>#</sup>,...,Liu X.*, Luo X.* & Liu, T.* (2022). Creation of a Yeast Strain with Co-Translationally Acylated Nucleosomes. <i>Angew. Chem. Int. Ed.</i></a>
                </h3></li>
                    <div class="recent-publication-picture">
                        <img src="images/WD 2022.jpg" height="200px"
                            alt="Figure">
                    </div>
                </div>

            <div class="publication">
                    <li><h3>
                        <a href="https://www.sciencedirect.com/science/article/pii/S2666166722002015"
                    target="_blank">Ling, X., Chang, L., Chen, H., Liu, T.* (2022). Efficient generation of locus-specific CAR-T cells with CRISPR/cCas12a. <i>Star Protocols.</i></a>
                    </h3></li>
                </div>


            <div class="publication">
                    <li><h3>
                        <a href="https://www.sciencedirect.com/science/article/pii/S0022283621006434"
                    target="_blank">Qin, X., Liu, T.* (2022). Recent Advances in Genetic Code Expansion Techniques for Protein Phosphorylation Studies. <i>Journal of Molecular Biology.</i></a>
                    </h3></li>
                </div>

            <div class="publication">
                    <li><h3>
                        <a href="https://www.sciencedirect.com/science/article/pii/S0022283622001395?via%3Dihub"
                    target="_blank">Liu, T.* (2022). Recent advances in Genetic Code Expansion: from cell engineering to protein design. <i>Journal of Molecular Biology.</i></a>
                    </h3></li>
                </div>

            <div class="publication">
                <li><h3><a href="https://pubs.acs.org/doi/full/10.1021/acscentsci.2c01142"
                    target="_blank"> Ling, X. & Liu, T.* (2022). Innovative CRISPR Screening Promotes Drug Target Identification. <i>ACS Central Science.</i></a>
                </h3></li>
                    <div class="recent-publication-picture">
                        <img src="images/2022 ACS CS.png" height="200px"
                            alt="Figure">
                    </div>
               </div>
            <div class="publication">
              <li><h3><a href="https://www.nature.com/articles/s41589-021-00899-z"
                   target="_blank"> Chen, C.<sup>#</sup>, Yu, G.<sup>#</sup>, Huang, Y.<sup>#</sup>, ... , Ye, H.* & Liu, T.* (2021). Genetic-code-expanded cell-based therapy for treating diabetes in mice. <i>Nature Chemical Biology.</i></a>
              </h3></li>
                <div class="recent-publication-picture">
                    <img src="images/2021 NCB.png" height="200px"
                         alt="Figure">
                </div>
            </div>

            <div class="publication">
                <li><h3>
                    <a href="https://www.nature.com/articles/s41589-021-00875-7"
                   target="_blank"> Song, X., Yang, F., ... Liu, T., ... Liu, X.*, Li, J.*, Cheng, J.* & Yao, X.* (2021). Dynamic crotonylation of EB1 by TIP60 ensures accurate spindle positioning in mitosis. <i>Nature Chemical Biology.</i></a>
                </h3></li>
            </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.sciencedirect.com/science/article/pii/S1097276521007735?via%3Dihub"
               target="_blank"> Ling, X.<sup>#</sup>, Chang, L.<sup>#</sup>, Chen, H., ... & Liu, T.* (2021). Improving the efficiency of CRISPR-Cas12a-based genome editing with site-specific covalent Cas12a-crRNA conjugates <i>Molecular Cells.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/34648747/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://pdf.sciencedirectassets.com/272198/AIP/1-s2.0-S1097276521007735/main.pdf?X-Amz-Security-Token=IQoJb3JpZ2luX2VjEMT%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIQCXShHbcYMZA%2BkNjDtXHHlx8ImHhocxi%2B4%2FIGezXqETpAIgKyM0plsuVeBsC0W1mf%2Fh1zSaB43I65HTizlHNwR2kxcq%2BgMIXBAEGgwwNTkwMDM1NDY4NjUiDFAXxIW4czoQ3SQ5FCrXAyXpyUEH4ezte3oY4DuDzsmJUCt6v4iFL%2F5vPzBaRhnJZFZTM6i29TLokbtVVP91S1NvsieiiXIzMapoONbFUsE1BNfQshke5ulZAVuIy5WkqT5vaVt2TE4r07JHTg1f7u2zkQu5c%2BhKSpl8Ub%2BcFyQKkkNYd4gt6g16LzTdxUvQBSTDYflzU8fAklxfi6p%2F9kxbQk2S92f%2B1vwngBC3sDvcxQv8Fi93yfdtoLXHGmHKVCx0ZEEDLYKaExXdavFxc5y0M1sry36J1CEIDJYS4%2Fj5FGh3znJSpHGC%2FwtpmmMBV8KeaYIfqEuPL49bk%2FNWFr3les%2B2gFrXHRGw5KCMG9FO90ZS2JC8%2Bb3rtbzZVTwgDioiRNjcC0bwgL69dLeQO%2BZBeWLknuYDiylSS9yMxg21awubkrDdgExWdv%2Bkm7S%2Fnv4Ad7md2nkxW2b4GktJJj9%2BTVon3K7CtEY4cMEqyP7Xd3uRX6GkZod4hPWtubcNgnIhdMagx5PxZRLW3QuqyOir8IsFba6m13fxIQrlXNCj%2F8O5GOX%2BwVAr%2FOKH0i6iQmlGk%2BBOcsja3mY1QYNMeloMeEtPYRcsp1uRnsf3oR6cToyNR9sMpwy7H9NsGgikDydz639WwzDAotqLBjqlARjSlIZElfQUdzVWPhRKBRbnDwF2ubuQVD0AYI7pVSU8wdVhdzDp0sn3tsS8T2x4cv%2FAYxdPnHcLWEH6ApSOq2iFr6dCa%2BJT0i%2FnT6ur1D9W1HKqBU6O1W3FhFtr0MOEmHEXdy5i87T6nnFGnqepjsUQwQ7Je0KQmmwfU0FjXesq660fDOO%2BwPxBkjqDx%2Brzsv4eM2WRlDQdUkDr%2FWZQZqL%2FD2ZwpQ%3D%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20211025T113114Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAQ3PHCVTYTCOHXI7X%2F20211025%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=3516337c8eec428dc5a6676cb5ee410886dbf6ef7f7accd001b8abfe66da8b2b&hash=df4168ab58e997be6c280cd030de2ae5490ac5995b06b45f55a29cc4e3066707&host=68042c943591013ac2b2430a89b270f6af2c76d8dfd086a07176afe7c76c2c61&pii=S1097276521007735&tid=spdf-f2f036e5-4369-49e3-8aa1-e80e2cb117d3&sid=7922d83d6e254247ec8b8c62675530e925dfgxrqa&type=client"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/mc 2021.jpg" height="300px"
                     alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <div class="abstract-content">
                    The CRISPR-Cas12a system shows unique features compared with widely used Cas9, making it an attractive and potentially more precise alternative. However, the adoption of this system has been hindered by its relatively low editing efficiency. Guided by physical chemical principles, we covalently conjugated 5′ terminal modified CRISPR RNA (crRNA) to a site-specifically modified Cas12a through biorthogonal chemical reaction. The genome editing efficiency of the resulting conjugated Cas12a complex (cCas12a) was substantially higher than that of the wild-type complex. We also demonstrated that cCas12a could be used for precise gene knockin and multiplex gene editing in a chimeric antigen receptor T cell preparation with efficiency much higher than that of the wild-type system. Overall, our findings indicate that covalently linking Cas nuclease and crRNA is an effective approach to improve the Cas12a-based genome editing system and could potentially provide an insight into engineering other Cas family members with low efficiency as well.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://chemistry-europe.onlinelibrary.wiley.com/doi/10.1002/cbic.202100357"
               target="_blank"> Cao, W., Qin, X., & Liu, T.* (2021). When supramolecular chemistry meets chemical biology: new strategies to target proteins through host–guest interactions <i>Chembiochem.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/34487417/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://chemistry-europe.onlinelibrary.wiley.com/doi/epdf/10.1002/cbic.202100357"
                       target="_blank">[PDF]</a>
                </div>
                <div class="recent-publication-picture">
                <img src="new pics/2021 CBC.png" height="200px"
                     alt="Figure">
                </div>
            </h3></li>
                <div class="abstract-content">
                    Supramolecular chemistry for targeting proteins is of great interest for the development of novel approaches to recognize, isolate and control proteins. Taking advantage of chemical biology approaches, such as genetic-code expansion and enzyme-mediated ligation, guest recognition elements can be built into proteins of interest, allowing supramolecular control of protein function and regulation. In this viewpoint article, we will discuss the methods, applications, limitations, and future perspectives of supramolecular chemistry for targeting proteins in a site-specific manner.
                </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://onlinelibrary.wiley.com/doi/10.1002/anie.202014540"
               target="_blank"> Wang, Y., Chen, X., Cai, W., Tan, L., Yu, Y., ... & Liu, T.* (2021). Expanding the structure diversity of protein building blocks with biosynthesized noncanonical amino acids from aromatic thiols. <i>Angewandte Chemie International Edition.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/33570250/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/anie.202014540"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/scheme 2021 angew WY.png" height="300px"
                     alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <div class="abstract-content">
                    Incorporation of structurally novel noncanonical amino acids (ncAAs) into proteins is valuable for both scientific and biomedical applications. To expand the structural diversity of available ncAAs and to reduce the burden of chemically synthesizing them, we have developed a general and simple biosynthetic method for genetically encoding novel ncAAs into recombinant proteins by feeding cells with economical commercially available or synthetically accessible aromatic thiols. We demonstrate that nearly 50 ncAAs with a diverse array of structures can be biosynthesized from these simple small‐molecule precursors by hijacking the cysteine biosynthetic enzymes, and the resulting ncAAs can subsequently be incorporated into proteins via an expanded genetic code. Moreover, we demonstrate that bio‐orthogonal reactive groups such as aromatic azides and aromatic ketones can be incorporated into green fluorescent protein or a therapeutic antibody with high yields, allowing for subsequent chemical conjugation, which will be useful both for research applications and for the development of homogeneously modified protein therapeutics.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://pubs.acs.org/doi/10.1021/acssynbio.0c00257"
               target="_blank">Hu, L., Qin, X., Huang, Y., ..., & Liu, T.* (2020). Thermophilic Pyrrolysyl-tRNA Synthetase Mutants for Enhanced Mammalian Genetic Code Expansion. <i>ACS Synthetic Biology.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/32931698/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://pubs.acs.org/doi/pdf/10.1021/acssynbio.0c00257"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/scheme of 2020 ACSSB.jpeg" height="200px"
                     alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <img src="new pics/scheme of 2020 ACSSB.jpg" height="200px"
                        alt="Cover">
                <div class="abstract-content">
                    Genetic code expansion (GCE) is a powerful technique for site-specific incorporation of noncanonical amino acids (ncAAs) into proteins in living cells, which is achieved through evolved aminoacyl-tRNA synthetase mutants. Stability is important for promoting enzyme evolution, and we found that many of the evolved synthetase mutants have reduced thermostabilities. In this study, we characterized two novel pyrrolysyl-tRNA synthetases (PylRSs) derived from thermophilic archaea: <i>Methanosarcina thermophila</i> (<i>Mt</i>) and <i>Methanosarcina flavescens</i> (<i>Mf</i>). Further study demonstrated that the wild-type PylRSs and several mutants were orthogonal and active in both <i>Escherichia coli</i> and mammalian cells and could thus be used for GCE. Compared with the commonly used <i>M. barkeri</i> PylRS, the wild-type thermophilic PylRSs displayed reduced GCE efficiency; however, some of the mutants, as well as some chimeras, outperformed their mesophilic counterparts in mammalian cell culture at 37 °C. Their better performance could at least partially be attributed to the fact that these thermophilic synthetases exhibit a threshold of enhanced stability against destabilizing mutations to accommodate structurally diverse substrate analogues. These were indicated by the higher melting temperatures (by 3–6 °C) and the higher expression levels that were typically observed for the <i>Mt</i>PylRS and <i>Mf</i>PylRS mutants relative to the <i>Mb</i> equivalents. Using histone H3 as an example, we demonstrated that one of the thermophilic synthetase mutants promoted the incorporation of multiple acetyl-lysine residues in mammalian cells. The enzymes developed in this study add to the PylRS toolbox and provide potentially better scaffolds for PylRS engineering and evolution, which will be necessary to meet the increasing demands for expanded substrate repertoire with better efficiency and specificity in mammalian systems.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.frontiersin.org/articles/10.3389/fbioe.2020.569191/full"
               target="_blank">Tan, L,. Zheng, Z., Xu, Y., ..., Liu, T.,* & Tang, H.* (2020). Efficient Selection Scheme for Incorporating Noncanonical Amino Acids Into Proteins in Saccharomyces cerevisiae. <i>Frontiers in Bioengineering and Biotechnology.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/33042970/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://www.frontiersin.org/articles/10.3389/fbioe.2020.569191/pdf"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/scheme 2020 FBB.jpg" height="300px"
                     alt="Figure">
                <div class="abstract-content">
                    With the advances in the field of expanded genetic code, the application of non-canonical amino acid (ncAA) is considered an effective strategy for protein engineering. However, cumbersome and complicated selection schemes limit the extensive application of this technology in <i>Saccharomyces cerevisiae</i>. To address this issue, a simplified selection scheme with confident results was developed and tested in this study. Based on a mutation library derived from <i>Escherichia coli</i> tyrosyl-tRNA synthetase (<i>Ec</i>TyrRS), a logic gate in synthetic biology was used to optimize screening procedures. We found that an “and” gate was more suitable than an “or” gate for isolating aminoacyl-tRNA synthetase from <i>S. cerevisiae</i>. The successful incorporation of <i>O</i>-methyltyrosine (OMeY) proved the utility and efficiency of this new selection scheme. After a round of positive selection, several new OMeY-tRNA synthetase (OMeYRS) mutants were screened, and their incorporation efficiency was improved. Furthermore, we characterized the insertion of several tyrosine analogs into Herceptine Fab and discovered that OMeYRS and its mutants were polyspecific. One of these mutants showed an optimal performance to incorporate different ncAAs into recombinant proteins in <i>S. cerevisiae</i>; this mutant was cloned and transfected into mammalian cells, and the results proved its functionality in HEK293 cells. This study could expand the application of ncAA in <i>S. cerevisiae</i> to construct efficient yeast cell factories for producing natural and synthetic products.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.nature.com/articles/s41392-020-00268-7" target="_blank">Huang, Y., & Liu, T.*(2020). Step further towards targeted senolytic therapy: therapeutic potential of uPAR-CAR T cells for senescence-related diseases. <i>Signal Transduction and Targeted Therapy.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/32792494/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://www.nature.com/articles/s41392-020-00268-7.pdf"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/scheme of 2020 STTT.jpg" height="300px"
                     alt="Figure">
                <div class="abstract-content">
                    A very recent study published in Nature by Amor <i>et al.</i> identifies the urokinase-type plasminogen activator receptor (uPAR) as a broadly induced specific cell-surface marker during cell senescence, which could be used as a target for chimeric antigen receptor (CAR) T cell to specifically ablate senescent cells <i>in vitro</i> and <i>in vivo</i>. The authors intriguingly show that uPAR-specific CAR T-cell therapy improves the treatment outcome of lung adenocarcinomas and efficiently reduces liver fibrosis in mouse models, thus offering a promising novel therapeutic strategy for senescence-associated diseases.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://onlinelibrary.wiley.com/doi/10.1002/anie.202009890" target="_blank">Zhang, Y.<sup>#</sup>, Ling, X.<sup>#</sup>, Su, X., ..., Liu, T., & Tang, X.* (2020). Optical Control of a CRISPR/Cas9 System for Gene Editing by Using Photolabile crRNA. <i>Angewandte Chemie International Edition.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/anie.202009890"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://pubs.rsc.org/en/content/articlehtml/2020/cc/d0cc01432j" target="_blank">Ling, X.<sup>#</sup>, Gao, X.<sup>#</sup>, ... & Liu, T.* (2020). Rational Design of Minimum CRISPR Guide RNA by Site-Specific Cas9-RNA Conjugation. <i>Chemical Communications.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/32510061/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://pubs.rsc.org/en/content/articlepdf/2020/cc/d0cc01432j"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/chem-comm.jpg" height="300px"
                     alt="Figure">
                <div class="abstract-content">
                    The CRISPR-Cas9 system enables facile and efficient genome engineering in living cells and organisms. We report a Cas9-RNA conjugation strategy to afford minimal crRNA containing only the guide sequence for the target gene, which may simplify and reduce the cost for large-scale and high-throughput crRNA synthesis and lead to new insights into the design of CRISPR family complexes for diverse purposes.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://advances.sciencemag.org/content/6/15/eaaz0051.abstract" target="_blank">Ling, X.<sup>#</sup>, Xie, B.<sup>#</sup>, Gao, X.<sup>#</sup>, ..., Li, M.,* & Liu T.* (2020).Improving the efficiency of precise genome editing with site-specific Cas9-oligonucleotide conjugates.&ensp;<i>Sciences Advances.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/32494588/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://advances.sciencemag.org/content/6/15/eaaz0051/tab-pdf"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/sci-adv.jpg" height="300px"
                     alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <img src="new pics/F1.medium.gif" height="300px"
                     alt="Cover">
                <div class="abstract-content">
                    Site-specific chemical conjugation of proteins can enhance their therapeutic and diagnostic utility but has seldom been applied to CRISPR-Cas9, which is a rapidly growing field with great therapeutic potential. The low efficiency of homology-directed repair remains a major hurdle in CRISPR-Cas9–mediated precise genome editing, which is limited by low concentration of donor DNA template at the cleavage site. In this study, we have developed methodology to site-specifically conjugate oligonucleotides to recombinant Cas9 protein containing a genetically encoded noncanonical amino acid with orthogonal chemical reactivity. The Cas9-oligonucleotide conjugates recruited an unmodified donor DNA template to the target site through base pairing, markedly increasing homology-directed repair efficiency in both human cell culture and mouse zygotes. These chemically modified Cas9 mutants provide an additional tool, one that is complementary to chemically modified nucleic acids, for improving the utility of CRISPR-Cas9–based genome-editing systems
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://pubs.acs.org/doi/10.1021/acs.biochem.9b00757" target="_blank">Qin, X., Tang, H., ..., & Liu T.*(2020). An Orthogonal Tyrosyl-tRNA Synthetase/tRNA Pair from a Thermophilic Bacterium for an Expanded Eukaryotic Genetic Code. <i>Biochemistry.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/31703481/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://pubs.acs.org/doi/pdf/10.1021/acs.biochem.9b00757"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/tyrRS.jpg" height="300px"
                     alt="Figure">
                <div class="abstract-content">
                    The Escherichia coli-derived tyrosyl-tRNA synthetase was the first enzyme engineered for genetic code expansion in a eukaryotic system but can charge only a limited set of structurally simple noncanonical amino acids. In contrast, the thermophilic Methanocaldococcus jannaschii-derived tyrosyl-tRNA synthetase mutants, used in only a prokaryotic system, can charge a surprisingly large set of structurally diverse ncAAs, due to their remarkable structural ability to tolerate mutations. Inspired by this, we characterized a new class of tyrosyl-tRNA synthetase/tRNATyr pairs from thermophilic bacterium Geobacillus stearothermophilus, which is homologous to the E. coli tyrosyl-tRNA synthetase but with better thermostability. This new pair is both orthogonal in mammalian cells and in Saccharomyces cerevisiae for genetic code expansion and can charge a diverse set of ncAAs with a comparable cellular efficiency, better specificity, and lower background, as compared to those of its E. coli homologue. This thermostable enzyme provides an alternative scaffold for synthetase library screening or evolution to genetically encode more structurally complex ncAAs in eukaryotic cells.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="http://article.cjcrcn.org/en/article/doi/10.21147/j.issn.1000-9604.2019.04.11"
                   target="_blank">Zhang, C., Ling, X., Guo, Y., ..., Liu, T., & Cui, H.* (2019). Evaluation of COC183B2 antibody targeting ovarian cancer by near-infrared fluorescence imaging. <i>Chinese Journal of Cancer Research.</i></a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/31564810/" target="_blank">[PubMed]</a>&ensp;&ensp;
                </div>
            </h3></li>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.sciencedirect.com/science/article/pii/S1001841719302463"
                   target="_blank">Ling, X., Chen, H., Zheng, W., Chang, L., Wang, Y., & Liu,T.* (2019).Site-specific protein modification by genetic encoded disulfide compatible thiols. <i>Chinese Chemical Letters.</i></a>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/xinyupaper.jpg" height="200px"
                     alt="Figure">
                <div class="abstract-content">
                    Cysteine chemistry provides a low cost and convenient way for site-specific protein modification. However, recombinant expression of disulfide bonding containing protein with unpaired cysteine is technically challenging and the resulting protein often suffers from significantly reduced yield and activity. Here we used genetic code expansion technique to introduce a surface exposed self-paired di-thiol functional group into proteins, which can be selectively reduced to afford active thiols. Two compounds containing self-paired disulfides were synthesized, and their genetic incorporations were validated using green fluorescent proteins (GFP). The compatibility of these self-paired di-thiols with natural disulfide bond was demonstrated using antibody fragment to afford site-specifically labeled antibody. This work provides another valuable building block into the chemical tool-box for site-specific labeling of proteins containing internal disulfides.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.sciencedirect.com/science/article/pii/S2405805X18300401?via%3Dihub"
                   target="_blank">Huang, Y., & Liu, T.* (2018). Therapeutic applications of genetic
                    code expansion. <i>Synthetic and Systems Biotechnology</i>.</a>
                <div class="pubmed-pdf">
                    <a href="https://pubmed.ncbi.nlm.nih.gov/30345400/" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://www.sciencedirect.com/science/article/pii/S2405805X18300401/pdfft?md5=b142c7eaaa9ebc1be3917bd715d087da&pid=1-s2.0-S2405805X18300401-main.pdf"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/2018-10.1016-j.synbio.2018.09.003-Fig.1.jpg" height="300px"
                     alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <img src="new pics/2018-10.1016-j.synbio.2018.09.003-cover.jpg" height="300px"
                     alt="Cover">
                <div class="abstract-content">
                    In nature, a limited, conservative set of amino acids are utilized to synthesize proteins. Genetic code expansion technique reassigns codons and incorporates noncanonical amino acids (ncAAs) through orthogonal aminoacyl-tRNA synthetase (aaRS)/tRNA pairs. The past decade has witnessed the rapid growth in diversity and scope for therapeutic applications of this technology. Here, we provided an update on the recent progress using genetic code expansion in the following areas: antibody-drug conjugates (ADCs), bispecific antibodies (BsAb), immunotherapies, long-lasting protein therapeutics, biosynthesized peptides, engineered viruses and cells, as well as other therapeutic related applications, where the technique was used to elucidate the mechanisms of biotherapeutics and drug targets.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://pubs.acs.org/doi/10.1021/jacs.8b06922" target="_blank">Tang, H.<sup>&#35;</sup>, Dai, Z.<sup>&#35;</sup>, Qin, X., Cai, W., Hu, L., Huang, Y., ... & Liu, T.* (2018). Proteomic identification of protein tyrosine phosphatase and substrate interactions in living mammalian cells by genetic encoding of irreversible enzyme inhibitors. <i>Journal of the American Chemical Society</i>.</a>
                <div class="pubmed-pdf">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/30247891" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://pubs.acs.org/doi/pdf/10.1021/jacs.8b06922"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/2018-10.1021-jacs.8b06922-Fig.1.jpeg" height="300px"
                     alt="Figure">
                <div class="abstract-content">
                    Protein tyrosine phosphatases (PTPs) play critical roles in cell signaling pathways, but identification of unknown PTPs for a given substrate in live cells remain technically challenging. Here, we synthesized a series of tyrosine-based irreversible PTP inhibitors and characterized by site-specific encoding on substrate proteins in cells with an expanded genetic code. By fine-tuning the chemical reactivity, we identified optimal active amino acid probes to covalently cross-link a PTP and its substrate both in vitro and in mammalian cells. Using HER2 as an example, we provide first direct evidence of HER2 Y1023 and SHP2 cross-linking in situ in living human cells. Moreover, proteomic analysis using our approach identified PTP1B as a novel phosphatase for HER2 that specifically dephosphorylated pY1221 position, which may shed light on the puzzle of PTP1B’s role in HER2 positive breast cancer. This novel method provides a useful tool for dissecting tyrosine phosphoregulation in living cells.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.cell.com/cell-chemical-biology/fulltext/S2451-9456(17)30180-0" target="_blank">Liu, T.*(Lead Contact), Jia, P., Ma, H., Reed, S. A., Luo, X., Larman, H. B., & Schultz, P. G.* (2017). Construction and Screening of a Lentiviral Secretome Library. <i>Cell Chemical Biology</i>, 24(6), 767-771.</a>
                <div class="pubmed-pdf">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28602759" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://www.cell.com/cell-chemical-biology/pdf/S2451-9456(17)30180-0.pdf"
                       target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="new pics/2017-10.1016-j.chembiol.2017.05-Fig1.jpg" height="300px"
                     alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <img src="images/2017-10.1016-j.chembiol.2017.05-cover.jpg" height="300px"
                     alt="Cover">
                <div class="abstract-content">
                    Over 2,000 human proteins are predicted to be secreted, but the biological function of the many of these proteins is still unknown. Moreover, a number of these proteins may act as new therapeutic agents or be targets for the development of therapeutic antibodies. To further explore the extracellular proteome, we have developed a secretome-enriched open reading frame (ORF) library that can be readily screened for autocrine activity in cell-based phenotypic or reporter assays. Next-generation sequencing (NGS) and database analysis predict that the library contains approximately 900 ORFs encoding known secreted proteins (accounting for 77.8% of the library), as well as genes encoding potentially unknown secreted proteins. In a proof-of-principle study, human TF-1 cells were screened for proliferative factors, and the known cytokine GMCSF was identified as a dominant hit. This library offers a relatively low-cost and straightforward approach for functional autocrine screens of secreted proteins.
                </div>
            </div>
        </div>
        <div class="publication">
            <li><h3>
                <a href="https://www.nature.com/articles/nchembio.2405" target="_blank">Luo, X., Fu, G., Wang, R. E., Zhu, X., Zambaldo, C., Liu, R., ... & Wang, F. (2017). Genetically encoding phosphotyrosine and its nonhydrolyzable analog in bacteria.. <i>Nature Chemical Biology</i>, 13(8), 845-849.</a>
                <div class="pubmed-pdf">
                    <a href="https://www.ncbi.nlm.nih.gov/pubmed/28604693" target="_blank">[PubMed]</a>&ensp;&ensp;
                    <a href="https://www.nature.com/articles/nchembio.2405.pdf" target="_blank">[PDF]</a>
                </div>
            </h3></li>
            <div class="publication-picture">
                <img src="images/2017-10.1038-nchembio.2405-Fig1.jpg" height="200px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                <img src="images/2017-10.1038-nchembio.2405-cover.gif" height="300px" alt="Cover">
                <div class="abstract-content">
                    Tyrosine phosphorylation is a common protein post-translational modification that plays a critical role in signal transduction and the regulation of many cellular processes. Using a propeptide strategy to increase cellular uptake of O-phosphotyrosine (pTyr) and its nonhydrolyzable analog 4-phosphomethyl-L-phenylalanine (Pmp), we identified an orthogonal aminoacyl-tRNA synthetase-tRNA pair that allows site-specific incorporation of both pTyr and Pmp into recombinant proteins in response to the amber stop codon in Escherichia coli in good yields. The X-ray structure of the synthetase reveals a reconfigured substrate-binding site, formed by nonconservative mutations and substantial local structural perturbations. We demonstrate the utility of this method by introducing Pmp into a putative phosphorylation site and determining the affinities of the individual variants for the substrate 3BP2. In summary, this work provides a useful recombinant tool to dissect the biological functions of tyrosine phosphorylation at specific sites in the proteome.
                </div>
            </div>
        </div>
    </div>
        <!-- After PKU End -->
        <!-- Before PKU -->
        <div class="publications" style="display: none" id="Before PKU">
            <h2>Before PKU</h2>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/10.1021/acschembio.6b00630" target="_blank">Liu, Y., Wang, Y., Zhang, Y., Liu, T., Jia, H., Zou, H., ... & Wang, F. (2016). Rational Design of Dual Agonist-Antibody Fusions as Long-acting Therapeutic Hormones. <i>ACS Chemical Biology</i>, 11(11), 2991-2995.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27704775" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/acschembio.6b00630" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2016-10.1021-acschembio.6b00630-Fig1.jpeg" height="250px"
                         alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2016-10.1021-acschembio.6b00630-cover.jpg" height="300px"
                         alt="Cover">
                    <div class="abstract-content">
                        Recent studies have suggested that modulation of two or more signaling pathways can achieve
                        substantial weight loss and glycemic stability. We have developed an approach to the generation
                        of bifunctional antibody agonists that activate leptin receptor and GLP-1 receptor. Leptin was
                        fused into the complementarity determining region 3 loop of the light chain alone, or in
                        combination with exendin-4 (EX4) fused at the N-terminus of the heavy chain of Herceptin. The
                        antibody fusions exhibit similar or increased in vitro activities on their cognate receptors,
                        but 50-100-fold longer circulating half-lives in rodents compared to the corresponding native
                        peptides/proteins. The efficacy of the leptin/EX4 dual antibody fusion on weight loss,
                        especially fat mass loss, was enhanced in ob/ob mice and DIO mice compared to the antibody
                        fusion of either EX4 or leptin alone. This work demonstrates the versatility of this
                        combinatorial fusion strategy for generating dual antibody agonists with long half-lives.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="http://www.pnas.org/content/113/21/5910.long" target="_blank">Liu, T., Wang, Y., Luo, X., Li, J., Reed, S. A., Xiao, H., ... & Schultz, P. G. (2016). Enhancing protein stability with extended disulfide bonds. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, 113(21), 5910-5915.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/27162342" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="http://www.pnas.org/content/pnas/113/21/5910.full.pdf" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2016-10.1073-pnas.1605363113-Fig1.jpg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2016-10.1073-pnas.1605363113-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        Disulfide bonds play an important role in protein folding and stability. However, the
                        cross-linking of sites within proteins by cysteine disulfides has significant distance and
                        dihedral angle constraints. Here we report the genetic encoding of noncanonical amino acids
                        containing long side-chain thiols that are readily incorporated into both bacterial and
                        mammalian proteins in good yields and with excellent fidelity. These amino acids can pair with
                        cysteines to afford extended disulfide bonds and allow cross-linking of more distant sites and
                        distinct domains of proteins. To demonstrate this notion, we preformed growth-based selection
                        experiments at nonpermissive temperatures using a library of random β-lactamase mutants
                        containing these noncanonical amino acids. A mutant enzyme that is cross-linked by one such
                        extended disulfide bond and is stabilized by ∼9 °C was identified. This result indicates that an
                        expanded set of building blocks beyond the canonical 20 amino acids can lead to proteins with
                        improved properties by unique mechanisms, distinct from those possible through conventional
                        mutagenesis schemes.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="http://www.pnas.org/content/113/13/3615.long" target="_blank">Luo, X., Zambaldo, C., Liu, T., Zhang, Y., Xuan, W., Wang, C., ... & Young, T. S. (2016). Recombinant thiopeptides containing noncanonical amino acids. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, 113(13), 3615-3620.</a>
                    <div class="pubmed-pdf"><a
                            href="https://www.ncbi.nlm.nih.gov/pubmed/26976568"
                            target="_blank">[PubMed]</a>&ensp;&ensp;<a
                            href="http://www.pnas.org/content/pnas/113/13/3615.full.pdf"
                            target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2016-10.1073-pnas.1602733113-Fig1.jpg" height="130px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2016-10.1073-pnas.1602733113-cover.jpg" height="230px" alt="Cover">
                    <div class="abstract-content">
                        Thiopeptides are a subclass of ribosomally synthesized and posttranslationally modified peptides
                        (RiPPs) with complex molecular architectures and an array of biological activities, including
                        potent antimicrobial activity. Here we report the generation of thiopeptides containing
                        noncanonical amino acids (ncAAs) by introducing orthogonal amber suppressor aminoacyl-tRNA
                        synthetase/tRNA pairs into a thiocillin producer strain of Bacillus cereus .We demonstrate that
                        thiopeptide variants containing ncAAs with bioorthogonal chemical reactivity can be further
                        postbiosynthetically modified with biophysical probes, including fluorophores and
                        photo-cross-linkers. This work allows the site-specific incorporation of ncAAs into thiopeptides
                        to increase their structural diversity and probe their biological activity; similar approaches
                        can likely be applied to other classes of RiPPs.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/anie.201507928" target="_blank">Luo, X.<sup>&#35;</sup>, Liu, T.<sup>&#35;</sup>, Wang, Y., Jia, H., Zhang, Y., Caballero, D., ... & Wang, F. (2015). An Epitope‐Specific Respiratory Syncytial Virus Vaccine Based on an Antibody Scaffold. <i>Angewandte Chemie International Edition.</i>, 54(48), 14531-14534.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/26434555" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/anie.201507928"
                           target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2015-10.1002-anie.201507928-Fig1.jpg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2015-10.1002-anie.201507928-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        Respiratory syncytial virus (RSV) is a leading cause of lower respiratory tract infections in
                        children. We have generated an epitope-specific RSV vaccine by grafting a neutralizing epitope
                        (F-epitope) in its native conformation into an immunoglobulin scaffold. The resulting antibody
                        fusion exhibited strong binding affinity to Motavizumab, an RSV neutralizing antibody, and
                        effectively induced potent neutralizing antibodies in mice. This work illustrates the potential
                        of the immunoglobulin molecule as a scaffold to present conformationally constrained B-cell
                        epitopes.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://www.sciencedirect.com/science/article/pii/S1367593115000708?via%3Dihub"
                       target="_blank">Liu, T.<sup>&#35;</sup>, Du, J.<sup>&#35;</sup>, Luo, X., Schultz, P. G., & Wang, F. (2015). Homogeneously modified immunoglobulin domains for therapeutic application. <i>Current Opinion in Chemical Biology</i>, 28(28), 66-74.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/26117722" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://www.sciencedirect.com/science/article/pii/S1367593115000708/pdfft?md5=bf8238af801d52d4b44af79a5fec17d4&pid=1-s2.0-S1367593115000708-main.pdf"
                           target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2015-10.1016-j.cbpa.2015.06-Fig1.jpg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2015-10.1016-j.cbpa.2015.06-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        The field of therapeutic antibodies has been revolutionized over the past decade, led by the
                        development of novel antibody-modification technologies. Besides the huge success achieved by
                        therapeutic monoclonal antibodies, a diversity of antibody derivatives have emerged with hope to
                        outperform their parental antibodies. Here we review the recent development of methodologies to
                        modify immunoglobulin domains and their therapeutic applications. The innovative genetic and
                        chemical approaches enable novel and controllable modifications on immunoglobulin domains,
                        producing homogeneous therapeutics with new functionalities or enhanced therapeutic profiles.
                        Such therapeutics, including antibody-drug conjugates, bispecific antibodies, and antibody/Fc
                        fusion proteins, have demonstrated great prospects in the treatment of cancer, auto-immune
                        diseases, infectious diseases, and many other disorders.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/cb501055h" target="_blank">Xiao, H., Xuan, W., Shao, S., Liu, T., & Schultz, P. G. (2015). Genetic Incorporation of &epsilon;-N-2-Hydroxyisobutyryl-lysine into Recombinant Histones. <i>ACS Chemical Biology</i>, 10(7), 1599-1603.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25909834" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/cb501055h" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2015-10.1021-cb501055h-Fig1.jpeg" height="250px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2015-10.1021-cb501055h-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        Here, we report the evolution of an orthogonal amber suppressor pyrrolysyl-tRNA synthetase
                        (PylRS)/tRNA<sub>CUA</sub><sup>Pyl</sup> pair that genetically encodes the post-translationally
                        modified amino acid, &epsilon;-N-2-hydroxyisobutyryl-lysine (HibK), in bacteria and mammalian
                        cells. HibK is a new type of histone mark that is widely distributed in histone proteins. The
                        ability to site-specifically incorporate HibK into proteins provides a useful tool to probe the
                        biological function of this newly identified post-translational modification.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/ja5130786" target="_blank">Liu, T., Fu, G., Luo, X., Liu, Y., Wang, Y., Wang, R. E., ... & Wang, F. (2015). Rational design of antibody protease inhibitors.. <i>Journal of the American Chemical Society</i>, 137(12), 4042-4045.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25775396" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/ja5130786" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2015-10.1021-ja5130786-Fig1.jpeg" height="250px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2015-10.1021-ja5130786-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        The bovine antibody BLV1H12, which has an ultralong CDR3H, provides a novel scaffold for
                        engineering new functions into the antibody's variable region. By modifying the &beta;-strand
                        "stalk" of BLV1H12 with sequences derived from natural or synthetic protease inhibitors, we have
                        generated antibodies that inhibit bovine trypsin and human neutrophil elastase (HNE) with low
                        nanomolar affinities. We were also able to generate a humanized variant using a human
                        immunoglobulin scaffold that shares a high degree of homology with BLV1H12. Further optimization
                        yielded a highly selective humanized anti-HNE antibody with sub-nanomolar affinity. This work
                        demonstrates a novel strategy for generating antibodies with potent and selective inhibitory
                        activities against extracellular proteases involved in human disease.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/jacs.5b00620" target="_blank">Wang, R. E.<sup>&#35;</sup>, Liu, T.<sup>&#35;</sup>, Wang, Y., Cao, Y., Du, J., Luo, X., ... & Schultz, P. G. (2015). An immunosuppressive antibody-drug conjugate.. <i>Journal of the American Chemical Society</i>, 137(9), 3229-3232.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25699419" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/jacs.5b00620" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2015-10.1021-jacs.5b00620-Fig1.jpeg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2015-10.1021-jacs.5b00620-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        We have developed a novel antibody-drug conjugate (ADC) that can selectively deliver the Lck
                        inhibitor dasatinib to human T lymphocytes. This ADC is based on a humanized antibody that
                        selectively binds with high affinity to CXCR4, an antigen that is selectively expressed on
                        hematopoietic cells. The resulting dasatinib-antibody conjugate suppresses T-cell-receptor
                        (TCR)-mediated T-cell activation and cytokine expression with low nM EC50 and has minimal
                        effects on cell viability. This ADC may lead to a new class of selective immunosuppressive drugs
                        with improved safety and extend the ADC strategy to the targeted delivery of kinase inhibitors
                        for indications beyond oncology.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="http://www.pnas.org/content/112/5/1356" target="_blank">Liu, T.<sup>&#35;</sup>, Zhang, Y.<sup>&#35;</sup>, Liu, Y.<sup>&#35;</sup>, Wang, Y., Jia, H., Kang, M., ... & Wang, F. (2015). Functional human antibody CDR fusions as long-acting therapeutic endocrine agonists. <i>Proceedings of the National Academy of Sciences of the United States of America</i>, 112(5), 1356-1361.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25605877" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="http://www.pnas.org/content/pnas/112/5/1356.full.pdf" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2015-10.1073-pnas.1423668112-Fig1.jpg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2015-10.1073-pnas.1423668112-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        On the basis of the 3D structure of a bovine antibody with a well-folded, ultralong
                        complementarity-determining region (CDR), we have developed a versatile approach for generating
                        human or humanized antibody agonists with excellent pharmacological properties. Using human
                        growth hormone (hGH) and human leptin (hLeptin) as model proteins, we have demonstrated that
                        functional human antibody CDR fusions can be efficiently engineered by grafting the native
                        hormones into different CDRs of the humanized antibody Herceptin. The resulting Herceptin CDR
                        fusion proteins were expressed in good yields in mammalian cells and retain comparable in vitro
                        biological activity to the native hormones. Pharmacological studies in rodents indicated a 20-
                        to 100-fold increase in plasma circulating half-life for these antibody agonists and
                        significantly extended in vivo activities in the GH-deficient rat model and leptin-deficient
                        obese mouse model for the hGH and hLeptin antibody fusions, respectively. These results
                        illustrate the utility of antibody CDR fusions as a general and versatile strategy for
                        generating long-acting protein therapeutics.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/ja5042447" target="_blank">Liu, T., Liu, Y., Wang, Y., Hull, M. V., Schultz, P. G., & Wang, F. (2014). Rational design of CXCR4 specific antibodies with elongated CDRs.. <i>Journal of the American Chemical Society</i>, 136(30), 10557-10560.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25041362" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/ja5042447" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2014-10.1021-ja5042447-Fig1.jpeg" height="250px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2014-10.1021-ja5042447-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        The bovine antibody (BLV1H12) which has an ultralong heavy chain complementarity determining
                        region 3 (CDRH3) provides a novel scaffold for antibody engineering. By substituting the
                        extended CDRH3 of BLV1H12 with modified CXCR4 binding peptides that adopt a &beta;-hairpin
                        conformation, we generated antibodies specifically targeting the ligand binding pocket of CXCR4
                        receptor. These engineered antibodies selectively bind to CXCR4 expressing cells with binding
                        affinities in the low nanomolar range. In addition, they inhibit SDF-1-dependent signal
                        transduction and cell migration in a transwell assay. Finally, we also demonstrate that a
                        similar strategy can be applied to other CDRs and show that a CDRH2-peptide fusion binds CXCR4
                        with a K<sub>d</sub> of 0.9 nM. This work illustrates the versatility of scaffold-based antibody
                        engineering and could greatly expand the antibody functional repertoire in the future.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/jm500743t" target="_blank">Qian, Z., Dougherty, P. G., Liu, T., Oottikkal, S., Hogan, P. G., Hadad, C. M., & Pei, D. (2014). Structure-based optimization of a peptidyl inhibitor against calcineurin-nuclear factor of activated T cell (NFAT) interaction.. <i>Journal of Medicinal Chemistry</i>, 57(18), 7792-7797.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25162754" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/jm500743t" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2014-10.1021-jm500743t-Fig1.jpeg" height="220px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2014-10.1021-jm500743t-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        Calcineurin inhibitors such as cyclosporine A and FK506 are effective immunosuppressants but
                        produce severe side effects. Rational modification of a previously reported peptide inhibitor,
                        GPHPVIVITGPHEE (K<sub>D</sub> &sim; 500 nM), by replacing the two valine residues with
                        <i>tert</i>-leucine and the C-terminal proline with a <i>cis</i>-proline analogue, gave an
                        improved inhibitor ZIZIT-<i>cis</i>Pro, which binds to calcineurin with a K<sub>D</sub> value of
                        2.6 nM and is more resistant to proteolysis.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/cb3005275" target="_blank">Qian, Z., Liu, T., Liu, Y. Y., Briesewitz, R., Barrios, A. M., Jhiang, S. M., & Pei, D. (2013). Efficient Delivery of Cyclic Peptides into Mammalian Cells with Short Sequence Motifs. <i>ACS Chemical Biology</i>, 8(2), 423-431.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/23130658" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/cb3005275" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2013-10.1021-cb3005275-Fig1.jpeg" height="230px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2013-10.1021-cb3005275-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        Cyclic peptides hold great potential as therapeutic agents and research tools, but their broad
                        application has been limited by poor membrane permeability. Here, we report a potentially
                        general approach for intracellular delivery of cyclic peptides. Short peptide motifs rich in
                        arginine and hydrophobic residues (e.g., F&Phi;RRRR, where &Phi; is <sub>L</sub>-2-naphthylalanine),
                        when embedded into small- to medium-sized cyclic peptides (7-13 amino acids), bound to the
                        plasma membrane of mammalian cultured cells and were subsequently internalized by the cells.
                        Confocal microscopy and a newly developed peptide internalization assay demonstrated that cyclic
                        peptides containing these transporter motifs were translocated into the cytoplasm and nucleus at
                        efficiencies 2-5-fold higher than that of nonaarginine (R<sub>9</sub>). Furthermore,
                        incorporation of the F&Phi;RRRR motif into a cyclic peptide containing a phosphocoumaryl
                        aminopropionic acid (pCAP) residue generated a cell permeable, fluorogenic probe for detecting
                        intracellular protein tyrosine phosphatase activities.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/cb200450w" target="_blank">Dewan, V.<sup>&#35;</sup>, Liu, T.<sup>&#35;</sup>, Chen, K. M., Qian, Z., Xiao, Y., Kleiman, L., ... & Musierforsyth, K. (2012). Cyclic peptide inhibitors of HIV-1 capsid-human lysyl-tRNA synthetase interaction.. <i>ACS Chemical Biology</i>, 7(4), 761-769.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/22276994" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/cb200450w" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2012-10.1021-cb200450w-Fig1.jpeg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2012-10.1021-cb200450w-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        The human immunodeficiency virus type 1 (HIV-1) capsid protein (CA) plays a critical role in the
                        viral life cycle. The C-terminal domain (CTD) of CA binds to human lysyl-tRNA synthetase
                        (hLysRS), and this interaction facilitates packaging of host cell tRNA<sup>Lys,3</sup>, which
                        serves as the primer for reverse transcription. Here, we report the library synthesis,
                        high-throughput screening, and identification of cyclic peptides (CPs) that bind HIV-1 CA.
                        Scrambling or single-residue changes of the selected peptide sequences eliminated binding,
                        suggesting a sequence-specific mode of interaction. Two peptides (CP2 and CP4) subjected to
                        detailed analysis also inhibited hLysRS/CA interaction <i>in vitro</i>. Nuclear magnetic
                        resonance spectroscopy and mutagenesis studies revealed that both CPs bind to a site proximal to
                        helix 4 of the CA-CTD, which is the known site of hLysRS interaction. These results extend the
                        current repertoire of CA-binding molecules to a new class of peptides targeting a novel site
                        with potential for development into novel antiviral agents.
                    </div>
                </div>
            </div>
			<div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/co200101w" target="_blank">Liu, T., Qian, Z., Xiao, Q., & Pei, D. (2011). High-throughput screening of one-bead-one-compound libraries: identification of cyclic peptidyl inhibitors against calcineurin/NFAT interaction.. <i>ACS Combinatorial Science</i>, 13(5), 537-546.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21848276" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/co200101w" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2011-10.1021-co200101w-Fig1.jpeg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2011-10.1021-co200101w-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        One-bead-one-compound (OBOC) libraries provide a powerful tool for drug discovery as well as
                        biomedical research. However, screening a large number of beads/compounds (>1 million) and rank
                        ordering the initial hits (which are covalently attached to a solid support) according to their
                        potencies still post significant technical challenges. In this work, we have integrated some of
                        the latest technical advances from our own as well as other laboratories to develop a general
                        methodology for rapidly screening large OBOC libraries. The methodology has been applied to
                        synthesize and screen a cyclic peptide library that features: (1) spatially segregated beads
                        containing cyclic peptides on the surface layer and linear encoding peptides in their interior;
                        (2) rapid on-bead screening of the library (>1 million) by a multistage procedure (magnetic bead
                        sorting, enzyme-linked assay, and fluorescence based screening); (3) selective release of cyclic
                        peptides from single positive beads for solution-phase determination of their binding
                        affinities; and (4) hit identification by partial Edman degradation/mass spectrometry (PED/MS).
                        Screening of the library against protein phosphatase calcineurin (Cn) identified a series of
                        cyclic peptides that bind to the substrate-docking site for nuclear factor of activated T cells
                        (NFAT) with K<sub>D</sub> values of ∼1 <i>&mu;</i>M. Further improvement of the affinity and
                        specificity of these compounds may lead to a new class of immunosuppressive agents that are more
                        selective and therefore less toxic than cyclosporine A and FK506.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/jm901778v" target="_blank">Liu, T., Liu, Y., Kao, H. Y., & Pei, D. (2010). Membrane permeable cyclic peptidyl inhibitors against human Peptidylprolyl Isomerase Pin1.. <i>Journal of Medicinal Chemistry</i>, 53(6), 2494-2501.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/20180533" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/jm901778v" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2010-10.1021-jm901778v-Fig1.jpeg" height="300px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2010-10.1021-jm901778v-cover.jpg" height="300px" alt="Cover">
                    <div class="abstract-content">
                        Peptidylprolyl isomerase Pin1 regulates the function and/or stability of phosphoproteins by
                        altering the conformation of specific pSer/pThr-Pro peptide bonds. In this work, a cyclic
                        peptide library was synthesized and screened against the catalytic domain of human Pin1. The
                        selected inhibitors contained a consensus motif of <sub>D</sub>-pThr-Pip-Nal (where Pip is
                        <sub>L</sub>-piperidine-2-carboxylic acid and Nal is <sub>L</sub>-2-naphthylalanine).
                        Representative compounds were tested for binding to Pin1 by isothermal titration calorimetry and
                        inhibition of Pin1 activity, and the most potent inhibitors had <i>K</i><sub>D</sub> (and
                        <i>K</i><sub>I</sub>) values in the low nanomolar range. Treatment of breast cancer cells with
                        the inhibitors, which were rendered membrane permeable by attachment of an octaarginine
                        sequence, inhibited cell proliferation and increased the protein levels of two previously
                        established Pin1 substrates, PML and SMRT. Finally, a second generation of cell permeable Pin1
                        inhibitors was designed by replacing the noncritical residues within the cyclic peptide ring
                        with arginine residues and shown to have antiproliferative activity against the cancer cells.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://www.sciencedirect.com/science/article/pii/S0968089608000175?via%3Dihub"
                       target="_blank">Liu, T., Joo, S. H., Voorhees, J. L., Brooks, C. L., & Pei, D. (2009). Synthesis and Screening of a Cyclic Peptide Library: Discovery of Small-Molecule Ligands against Human Prolactin Receptor. <i>Bioorganic & Medicinal Chemistry</i>, 17(3), 1026-1033.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/18234500" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://www.sciencedirect.com/science/article/pii/S0968089608000175/pdfft?md5=6e082a9906e661a9d30f603fc933aaf5&pid=1-s2.0-S0968089608000175-main.pdf"
                           target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2009-10.1016-j.bmc.2008.01-Fig1.jpg" height="150px" alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2009-10.1016-j.bmc.2008.01-cover.gif" height="200px" alt="Cover">
                    <div class="abstract-content">
                        Prolactin receptor is involved in normal lactation and reproduction; however, excessive
                        prolactin levels can cause various reproductive disorders such as prolactinomas. Small-molecule
                        antagonists against the human prolactin receptor (hPRLr) thus have potential clinical
                        applications and may serve as useful molecular probes in biomedical research. In this work, we
                        synthesized a large, support-bound cyclic peptide library (theoretical diversity of
                        1.2x10<sup>7</sup>) on 90-&mu;m TentaGel beads and screened it against the extracellular domain
                        of hPRLr. To facilitate hit identification, each TentaGel bead was spatially segregated into
                        outer and inner layers, with a cyclic peptide displayed on the bead surface while the bead
                        interior contained the corresponding linear peptide. The identity of a positive bead was
                        revealed by sequencing the linear encoding peptide within the bead by partial Edman
                        degradation/mass spectrometry. Screening of the library resulted in 20 hits, two of which were
                        selected for further analysis and shown to bind to hPRLr with dissociation constants of 2-3 &mu;M.
                    </div>
                </div>
            </div>
			<div class="publication">
                <li><h3><a href="https://link.springer.com/article/10.1007/s11434-006-0530-9" target="_blank">Liu, T., Chen, L., Hou, S., & Feng, X. (2006). Non-viral gene delivery carrier of probe type host molecule—Interactions between DNA and β-cyclodextrin derivative complexes (I). <i>CHINESE SCI BULL</i> 2006 51: 530. </a></h3></li>
                <div class="publication-picture">
                    <img src="images/2006-10.1007-s11434-006-0530-9-Fig1.png" height="200px"
                         alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2006-10.1007-s11434-006-0530-9-cover.jpg" height="200px"
                         alt="Cover">
                    <div class="abstract-content">
                        A host type non-virus gene delivery carrier, phenanthroline-&beta;-cyclodextrin derivative host
                        molecule, was produced which can be used as molecular probe. Interactions between DZY-1 and DNA
                        were investigated by electrophoresis assay. <i>Hind</i> III enzyme inhibition assay was carried
                        out using DNA condensates induced by host molecules or host-guest molecule complexes to explore
                        their ability to inhibit enzyme digestion. Micro-structure of DNA condensates induced by host
                        molecules and host-guest molecule complexes was observed by scanning electron microscope (SEM).
                        Our work indicates the delivery mechanism of DZY-1 used as a gene delivery carrier and also
                        provides a method to design and produce non-virus gene delivery carriers.
                    </div>
                </div>
            </div>
            <div class="publication">
                <li><h3>
                    <a href="https://pubs.acs.org/doi/full/10.1021/bc034230p" target="_blank">Liu, Y., Chen, G. S., Chen, Y., Ding, F., Liu, T., & Zhao, Y. (2004). Molecular binding behavior of pyridine-2,6-dicarboxamide-bridged bis(<i>&beta;</i>;-cyclodextrin) with oligopeptides : switchable molecular binding mode. <i>Bioconjugate Chemistry</i>, 15(2), 300-306.</a>
                    <div class="pubmed-pdf">
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/15025525" target="_blank">[PubMed]</a>&ensp;&ensp;
                        <a href="https://pubs.acs.org/doi/pdf/10.1021/bc034230p" target="_blank">[PDF]</a>
                    </div>
                </h3></li>
                <div class="publication-picture">
                    <img src="images/2004-10.1021-bc034230p-Fig1.gif" height="250px"
                         alt="Figure">&ensp;&ensp;&ensp;&ensp;
                    <img src="images/2004-10.1021-bc034230p-cover.gif" height="250px"
                         alt="Cover">
                    <div class="abstract-content">
                        Bridged bis(<i>&beta;</i>-cyclodextrin) 1 with a pyridine-2,6-dicarboxamide linker was
                        synthesized, and its inclusion complexation behavior with some aliphatic oligopeptides was
                        investigated in aqueous buffer solution of pH 2.0 and 7.2 at 25 degrees C by means of circular
                        dichroism, fluorescence, and 2D NMR techniques. The results show that the resulting inclusion
                        complexes of 1 with oligopeptides adopt a cooperative "cyclodextrin-guest-cyclodextrin" sandwich
                        binding mode in a neutral media, but a "guest-linker-cyclodextrin" coinclusion binding mode in
                        an acidic media. These switchable binding modes consequently rationalize the binding ability of
                        bis(<i>&beta;</i>-cyclodextrin) 1 at different pH values; that is, 1 shows the stronger
                        association with oligopeptides in a neutral media. Because of the simultaneous contributions of
                        hydrophobic, hydrogen bond, and electrostatic interactions, bis(<i>&beta;</i>-cyclodextrin) 1
                        affords length-selectivity up to 4.7 for the Gly-Gly/Gly-Gly-Gly pair at pH 2.0 and
                        sequence-selectivity up to 4.2 for the Gly-Leu/Leu-Gly pair at pH 7.2. These phenomena are
                        discussed from the viewpoint of the size-fit concept and the multipoint recognitions between
                        host and guest.
                    </div>
                </div>
            </div>
        </div>
        <!-- Before PKU End -->
	    </ol>
    </div>
    <!-- Publications End -->
</div>
<script>
    function openYear(evt, years) {
        var i, tablinks;
        var x = document.getElementsByClassName("publications");
        for (i = 0; i < x.length; i++) {
            x[i].style.display = "none";
        }
        tablinks = document.getElementsByClassName("tablink");
        for (i = 0; i < x.length; i++) {
            tablinks[i].className = tablinks[i].className.replace("publication-sidebar-item-active", "publication-sidebar-item");
        }
        document.getElementById(years).style.display = "block";
        evt.currentTarget.className = evt.currentTarget.className.replace("publication-sidebar-item", "publication-sidebar-item-active");
    }
</script>
<br/>
<!-- Footer -->
<div id="footer-container"></div>
<!-- Footer End -->
<script src="js/components.js"></script>
<script src="js/common.js"></script>
</body>
</html>
